spec/syntax.md

```diff
@@ -108,7 +109,7 @@ let hello = new MessageFormat('{Hello, world!}')
 hello.format()
 ```
 
-### Expressions
+### Placeholder Expressions
```

Comment:
As discussed at our last meeting, an expression is also used for variable definitions and for selectors; not just placeholders. I think changing that terminology should be beyond the scope of this PR.

Reply:
I made the links work in the spec doc by expanding the term here. Note that the term "placeholder" doesn't appear in the ABNF and is not currently defined anywhere (else). The syntax of an expression includes the `{`/`}` markers and so I concluded that it was reasonable to say that all expressions are placeholders. The text recognizes that placeholder expressions can appear in variable definitions and selectors.

The easier way to fix this would be to use singular instead of plural and I'll do that for now.